infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSwitchController.kt

```diff
@@ -77,7 +77,7 @@ class LayoutSwitchController(
         @RequestParam("ids", required = true) ids: List<IntId<TrackLayoutSwitch>>,
     ): List<TrackLayoutSwitch> {
         logger.apiCall("getTrackLayoutSwitches", "ids" to ids, "publishType" to publishType)
-        return ids.mapNotNull { id -> switchService.get(publishType, id) }
+        return switchService.getMany(publishType, ids)
```

Comment:
Tämä ja alempi olikin käytännössä ainoat nykyiset paikat jossa näitä massahakuja oli sellaisenaan käytetty. Nyt kun on sekä cachetus että mekanismi näiden käyttämiseen fiksummin, niin näitä voisi alkaa viljelemään yleisemmin `Promise.all`:in sijasta